buchschloss/gui2/actions.py

```diff
@@ -154,11 +136,8 @@ def view_wrap(*args, **kwargs):
 
     search_frame = tk.Frame(master)
     search_frame.pack()
-    rw = widgets.SearchResultWidget(search_frame, results, view_wrap)
+    rw = widgets.SearchResultWidget(search_frame, tuple(results), view_wrap)
```

Comment:
is this still needed?

Reply:
``len()`` needed, so it's here or in ``__init__``. Or ``seach`` could use a list comprehension.

Reply:
Since it's conceivable someone may want only the first result (``utils.get_book_data``), I'll leave it here.